site/content/contribute/getting-started/labels.md

```diff
@@ -0,0 +1,58 @@
+* `First Good Issue`: Suitable for first-time contributors.
+* `Framework/<name>`: Requires using the named framework (ReactJS, Redux, etc.)
+* `Help Wanted`: Community help wanted.
+* `Language/<name>`: Requires writing in the named language (Go, JavaScript, etc.)
+* `Localization`: Relates to translating the user interface.
+* `Move to Feature Ideas forum`: Marked for relocation to the feature ideas forum.
+* `Move to Troubleshooting`: Marked for relocation to the troubleshooting section of the documentation.
+* `PR Submitted`: A pull request has been opened for this issue.
+* `Repository/<name>`: Requires changes in the named respository.
+* `Up for Grabs`: Ready for help from the community. Removed when someone volunteers.
+
+# Pull Request Labels
+
+* `1: PM Review`: Requires review by a [product manager](/contribute/getting-started/core-committers/#product-managers).
+* `2: Dev Review`: Requires review by a [core commiter](/contribute/getting-started/core-committers/#core-committers).
+* `3: Ready to Merge`: All reviewers have approved the pull request.
+* `Awaiting Submitter Action`: Blocked on the author.
+* `Changelog/Done`: Required changelog entry has been written.
+* `Changelog/Not Needed`: Does not require a changelog entry.
+* `CherryPick/Approved`: Meant for the quality or patch release tracked in the milestone.
+* `CherryPick/Candidate`: A candidate for a quality or patch release, but not yet approved.
+* `CherryPick/Done`: Successfully cherry-picked to the quality or patch release tracked in the milestone.
+* `Do Not Merge`: Should not be merged until this label is removed.
+* `Do Not Merge/Awaiting PR`: Awaiting another pull request before merging (e.g. server changes).
+* `Do Not Merge/Awaiting loadtest`: Must be loadtested before it can be merged.
+* `Do Not Merge/Awaiting next release`: To be merged with the next release (e.g. API documentation updates).
+* `Docs/Done`: Required documentation has been written.
+* `Docs/Needed`: Requires documentation.
+* `Docs/Not Needed`: Does not require documentation.
+* `Hackfest`: Related to a Mattermost hackathon.
+* `Hacktoberfest`: Related to [Hacktoberfest](https://hacktoberfest.digitalocean.com/).
+* `Lifecycle/1:stale`: Inactive for 14 days.
+* `Lifecycle/2:inactive`: Inactive for 28 days.
+* `Lifecycle/3:orphaned`: Inactive for 58 days, and may be closed.
+* `Lifecycle/frozen`: Ignores the normal lifecycle flow.
+* `Loadtest`: Triggers an automatic load test.
+* `Major Change`: The pull request is a major feature or affects large areas of the code base (e.g. [moving channel store and actions to Redux](https://github.com/mattermost/platform/pull/6235)).
+* `Need submitted action to join approved contributor list`
+* `QA Review`: Requires review by a [QA tester](/contribute/getting-started/core-committers/#qa-testers).
+* `Setup Test Server`: Triggers the creation of a test server.
+* `Setup Upgrade Test Server`: Triggers the creation a test server and performs an upgrade.
```

Comment:
Does this label still do this?

Reply:
Good question: @cpanato, is the `Setup Upgrade Test Server` still "a thing"?